backend/app/routers/dependencies.py

```diff
@@ -1,3 +1,9 @@
-def next_records_parameters(record_key: str, record_type: str) -> dict[str, str]:
+from app.models.types import RecordType
+
+
+def next_records_parameters(
+    record_key: str, record_type: RecordType
+) -> dict[str, str | RecordType]:
```

Comment:
Suggest a Pydantic model here.

Reply:
Yes, I removed this function and replaced it with a Pydantic model.